books/books.py

```diff
@@ -0,0 +1,54 @@
+import requests
+from bs4 import BeautifulSoup
+import os
+
+url = "http://gen.lib.rus.ec/search.php"
+search = (input("Enter the name of the book: "))
+r = requests.get(url, {'req':search})
+url = r.url + "&lg_topic=libgen&open=0&view=simple&res=100&phrase=1&column=def"
```

Comment:
Instead of doing this like this, you could have given all of these in request parameters like the 'search'